# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hotplug++/no-isapnp.h.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Not in latest Linux headers anymore.

  - Rene Rebe <rene@exactcode.de>

--- hotplug++-0.1.1/bin/isapnp-match.hh	2005-09-21 09:28:06.000000000 +0000
+++ hotplug++-0.1.1-new/bin/isapnp-match.hh	2007-01-28 19:43:24.000000000 +0000
@@ -3,7 +3,13 @@
 #include <match.hh>
 
 #define ISAPNP_CARD_DEVS	8
-#include <linux/isapnp.h>
+#define ISAPNP_VENDOR(a,b,c)    (((((a)-'A'+1)&0x3f)<<2)|\
+                                ((((b)-'A'+1)&0x18)>>3)|((((b)-'A'+1)&7)<<13)|\
+                                ((((c)-'A'+1)&0x1f)<<8))
+#define ISAPNP_DEVICE(x)        ((((x)&0xf000)>>8)|\
+                                 (((x)&0x0f00)>>8)|\
+                                 (((x)&0x00f0)<<8)|\
+                                 (((x)&0x000f)<<8))
 
 class isapnp_module {
 public:
